src/REPLMode.jl

```diff
@@ -170,9 +170,10 @@ end
 
 # packages can be identified through: uuid, name, or name+uuid
 # additionally valid for add/develop are: local path, url
-function parse_package(word::AbstractString; add_or_develop=false)::PackageSpec
+function parse_package(word::AbstractString; add_or_develop::Bool=false)::PackageSpec
     word = replace(word, "~" => homedir())
     if add_or_develop && casesensitive_isdir(word)
+        @info "Resolving package specifier `$word` as a directory at `$(abspath(word))`."
```

Comment:
`Resolving` -> `resolving` and `abspath(word)` -> `Base.contractuser(abspath(word))`?

Reply:
good call :+1: